### ISSUE TITLE
Add icons for a few more sites

### ISSUE THREAD

**Issue by snarfed** [dshanske/syndication-links#182]
Hi @dshanske! Any chance we could get icons for these sites?

- https://news.indieweb.org/
- https://bsky.app/ (ideally *.bsky.app + *.bsky.social)
- https://fed.brid.gy/
- ~~https://substack.com/~~
- ~~https://stackoverflow.com/~~

Low priority, definitely lower than everything you all are currently working on in the webmention plugin. Thanks in advance!

(Related: #38, #99)

**Comment by dshanske** [comment]
Can you provide the icon links in SVG

**Comment by snarfed** [comment]
I don't have them handy yet, no

**Comment by dshanske** [comment]
@snarfed Substack and Stack Overflow are in the icon set.

**Comment by drikkes** [comment]
Bluesky would be nice. But I have no idea, how this icon could look like. The app has no real logo.

**Comment by snarfed** [comment]
True! A number of people have come up with popular unofficial logos:
* cloud: https://staging.bsky.app/profile/vito.fyi/post/3jqkauv6krk2y
* abstract butterfly: https://staging.bsky.app/profile/vito.fyi/post/3jt7sjbjcpi2z
* less abstract butterfly: https://staging.bsky.app/profile/birdapp.rip/post/3jt7hpz3jwb2t

I used the abstract butterfly on both https://bluesky-atom.appspot.com and the top right of https://snarfed.org/ . @dshanske here are the files:
https://github.com/snarfed/bluesky-atom/tree/main/static
https://github.com/snarfed/snarfed.org/commit/8be47f087ea85aa2ea112abd64bb77f686a9a3af

**Comment by snarfed** [comment]
Bridgy's is https://github.com/snarfed/bridgy/blob/main/static/favicon.png , feel free to use that for both Bridgy Fed and Bridgy classic.

**Comment by drikkes** [comment]
Hi. Any progress on this? Personally, I'd go with the simple cloud symbol from Genericons Neue, but I have no idea, how to add that. For sure not in a way that automatically detects an added link is from Bluesky.

**Comment by dshanske** [comment]
I'm happy to put in a placeholder if no one has a specific icon then

**Comment by snarfed** [comment]
Um, @drikkes and I both had specific icon suggestions above, right?

**Comment by drikkes** [comment]
Although I don't really get why a butterfly, I'd prefer it to the default globe icon.

**Comment by dshanske** [comment]
Yes, but they weren't svg, but I'll see what I can figure out

**Comment by drikkes** [comment]
As said, the Genericon would do.
`<svg class="genericons-neue genericons-neue-cloud" width="16px" height="16px"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="genericons-neue.svg#cloud"></use></svg>`

**Comment by snarfed** [comment]
related: https://github.com/bluesky-social/atproto/discussions/1754

**Comment by dshanske** [comment]
I added in a blue cloud icon for Bluesky as a placeholder for now. Indieweb News has the  indieweb logo. 

With Bridgy, still need something in the right format to represent.

**Comment by drikkes** [comment]
With the butterfly @snarfed backed the right horse. Someone opened another issue for it.
